papaparse.js

```diff
@@ -426,7 +426,6 @@
 	function ChunkStreamer(config)
 	{
 		this._handle = null;
-		this._paused = false;
```

Comment:
Per https://github.com/mholt/PapaParse/pull/166/files#r28797862, this variable is never used.